### PR TITLE
webedit health analyzer rmb fix

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
health analyzer rmb works again because someone forgot to fix that when patching out the cell thing
## Why It's Good For The Game
oversight fix good
## Proof of Testing
my brother in christ it is a webedit if you really want me to test it i'll do it but i'll be mildly disgruntled by it  

## Changelog

:cl:
fix: health analyzer rmb works again
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/scanners/health_analyzer.dm [hunk 1]
```diff
@@ -82,7 +82,7 @@
 	add_fingerprint(user)
 
 /obj/item/healthanalyzer/attack_secondary(mob/living/victim, mob/living/user, params)
-	if(!user.can_read(src) || !(item_use_power(power_use_amount, user) & COMPONENT_POWER_SUCCESS)) // SKYRAT EDIT CHANGE: Blind People Can Analyze Again/Power Cost
+	if(!user.can_read(src))
```
Comment (tf-4) [inline code/game/objects/items/devices/scanners/health_analyzer.dm]: // SKYRAT EDIT CHANGE: Blind People Can Analyze Again

Comment (tf-4) [reply code/game/objects/items/devices/scanners/health_analyzer.dm]: pretend thats a suggestion

Comment (tf-4) [inline code/game/objects/items/devices/scanners/health_analyzer.dm]:
```suggestion
	//if(!user.can_read(src) || user.is_blind()) - ORIGINAL
	if(!user.can_read(src)) // SKYRAT EDIT CHANGE - Blind people can analyse again
```
Actually do it like this

**Comment by YakumoChen** [comment]
yeah uh my bad I skipped over it thinking uh blind people lines mentally because I thought it was a different PR

**Review by tf-4 — APPROVED** [review]

**Pull request merged**